Gulpfile.js

```diff
@@ -1,49 +1,34 @@
-var gulp = require("gulp");
-var sourcemaps = require("gulp-sourcemaps");
-var babel = require("gulp-babel");
-var concat = require("gulp-concat");
-var uglify = require("gulp-uglify");
-var browserify = require('browserify');
-var through2 = require('through2');
-var babelify = require('babelify');
+var gulp = require('gulp');
+var babel = require('gulp-babel');
+var webpack = require('webpack');
 
-gulp.task('buildWeb',function(){
-	return gulp.src('./src/main.js')
-		.pipe(sourcemaps.init())
-		.pipe(through2.obj(function(file,enc,next){
-			browserify(file.path,{debug: process.env.NODE_ENV === 'development'})
-				.transform(babelify, {
-					presets: ["es2015"],
-					plugins: ["babel-plugin-add-module-exports"]
-				})
-				.bundle(function (err, res){
-					if(err){
-						return next(err);
-					}
-					file.contents = res;
-					next(null,file);
-				});
-		}))
-		.on('error', function(error){
-			console.log(error.stack);
-			this.emit('end');
-		})
-		.pipe(concat("meyda.js"))
-		.pipe(gulp.dest('./dist/web/'))
-		.pipe(uglify())
-		.pipe(concat("meyda.min.js"))
-		.pipe(sourcemaps.write('.'))
-		.pipe(gulp.dest("./dist/web"));;
+gulp.task('webpack', function(callback) {
+	webpack(require('./webpack.config.js').default,
+		function(err, stats) {
+			if(err) throw new Error("webpack", err);
+			callback();
+		}
+	);
 });
 
+gulp.task('webpack-minify', ['webpack'], function(callback) {
```

Comment:
Yeah sounds fair, I don't actually care about codeclimate personally, @hughrawlinson what do you think?

I'd say given that this fixes our /currently/ broken library, it might be worth ignoring CC for now